quick-start-guide/pom.xml

```diff
@@ -18,6 +18,20 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
+    <parent>
+        <groupId>org.wso2.samples.is</groupId>
+        <artifactId>samples-is</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+    <dependency>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+    </dependency>
+    </dependencies>
+
     <modelVersion>4.0.0</modelVersion>
     <groupId>Quick-Start-Guide</groupId>
```

Comment:
Group Id needs to be changed (should be removed so that it will use the parent's one). And artifact id also should match the convention.